### PR TITLE
add Sentio Debugger page

### PR DESCRIPTION
**Pull request by meganskye**

### CODE REVIEW

**Comment by Maar-io** [comment]
The last 5 images do not need to be full width

**Comment by meganskye** [comment]
> The last 5 images do not need to be full width

tweaked image sizes in new commit

File: docs/build/integrations/indexers/sentio.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+
+**Direct Build**
+
+You can also click the simulator button on the left navigation bar and go to the simulator page which shows all the history simulations. Click the simulation button on the right corner will pop a similar UI but without prepopulated transaction data.
+
+<Figure src={require('./img/sentio19.png').default } width="85%" />
+
+**Override Contract**
+
+Use the compilations tab to upload a local contract compilation, folder.
```
Comment (gluneau) [inline docs/build/integrations/indexers/sentio.md:189]: Is the comma extra or maybe there is some missing words?

File: docs/build/integrations/indexers/sentio.md [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+
+<Figure src={require('./img/sentio14.png').default } width="100%" />
+
+The debugger also supports adding **user defined watched variables (similar to a regular debugger.)**
+
+<Figure src={require('./img/sentio15.png').default } width="100%" />
+
+**Limitations**
+
+- Contracts compiled with viaIR option are not fully supported.
```
Comment (gluneau) [inline docs/build/integrations/indexers/sentio.md:140]: Is `viaIR` a special word, or is it missing a space?

**Review by gluneau — CHANGES_REQUESTED** [review]

File: docs/build/integrations/indexers/sentio.md (hunk 2 above)
Comment (meganskye) [reply docs/build/integrations/indexers/sentio.md:140]: viaIR enables the new Yul-based compiler pipeline for Solidity

File: docs/build/integrations/indexers/sentio.md (hunk 1 above)
Comment (meganskye) [reply docs/build/integrations/indexers/sentio.md]: nice catch! thanks!

**Review by gluneau — APPROVED** [review]

**Pull request merged**